pkg/api/vlabs/types.go

```diff
@@ -226,32 +254,6 @@ type KeyVaultCertificate struct {
 	CertificateStore string `json:"certificateStore,omitempty"`
 }
 
-// OrchestratorType defines orchestrators supported by ACS
-type OrchestratorType string
```

Comment:
I see. custom type is good for type-checking but if it is more cumbersome than beneficial here, then we don't need the custom type. There are other custom types such as OsType too. We can leave it for now.